src/main/java/mods/railcraft/common/blocks/aesthetics/brick/BrickTheme.java

```diff
@@ -52,8 +52,7 @@ public void initRecipes(BlockBrick block) {
     BLEACHEDBONE(RailcraftBlocks.BRICK_BLEACHED_BONE, MapColor.ADOBE) {
         @Override
         public void initRecipes(BlockBrick block) {
-            CraftingPlugin.addShapelessRecipe(RailcraftItems.BLEACHED_CLAY.getStack(), new ItemStack(Items.CLAY_BALL), new ItemStack(Items.DYE, 1, 15), new ItemStack(Items.DYE, 1, 15), new ItemStack(Items.DYE, 1, 15));
-            CraftingPlugin.addFurnaceRecipe(RailcraftItems.BLEACHED_CLAY.getStack(), new ItemStack(block, 1, 2), 0.3F);
+            CraftingPlugin.addFurnaceRecipe(new ItemStack(Blocks.BONE_BLOCK), new ItemStack(block, 1, 2), 0.3F);
```

Comment:
Bones only become cheap once you build some kind of mob farm, and even then, unless its an especially cheesy, tps breaking one, the output isn't that great.

Doubling the output would be fine.